sig-architecture/production-readiness.md

```diff
@@ -28,6 +27,19 @@ aspects of the process.
     happens if it is subsequently upgraded again?
   - Are there tests for this?
 * Scalability
+  - Does enabling/using the feautre will result in new API calls?
+    Keeping in mind the [supported limits] (e.g. 5000 nodes per cluster,
+    100 pods/s churn), what QPS is expected for each of those?
+  - Does enabling/using the feature will result in supporting new API
+    objects? How many of those will be supported (and how that translates
+    to limitations for users)?
+  - Does enabling/using the feature will result in increasing size of
+    the existing API objects? How much?
+  - Does enabling/using the feature will result in increasing latencies
+    of any operations covered by [existing SLIs/SLOs] (e.g. by adding
+    additional work)? Please describe the details about impact if so.
+  - Does enabling/using the feature will result in non-negligible increase
+    of resource usage (CPU, RAM, disk IO, ...) in any components?
```

Comment:
Maybe some question about scaling resources of any new components (if all new components will adjust resource requests based on the cluster size)?